hypothesis-python/src/hypothesis/extra/pandas/impl.py

```diff
@@ -126,6 +126,8 @@ def convert_element(value):
             if is_na_dtype and value is None:
                 return None
             name = f"draw({prefix}elements)"
+            if dtype.kind == "O":
+                return value  # for objects, just use the object
```

Comment:
for my understanding: this is a fast path, and equivalent to going through `return np.array([value], dtype=dtype)[0]` below? Could numpy ever raise eg `OverflowError` on `dtype="O"`? (probably not).